doc/sr_cpp_walkthrough.rst

```diff
@@ -69,8 +67,8 @@ executes a preprocessing script.
 Scripts
 =======
 
-The example in :ref:`SR CPP Models` shows how to store, and use a PyTorch script
-in the database with the C++ Client.  The script is stored a file
+The example in :ref:`SR CPP Models` shows how to store and use a PyTorch script
+in the database with the C++ Client.  The script stores as a file
```

Comment:
The script is the file we're talking about here, so while you were right to notice that this text needed updating, I think the correction should be `is stored as`